Sources/Database+MySQL.swift

```diff
@@ -19,7 +19,7 @@ extension Database {
         childTable: String,
         childForeignKey: String
         ) -> String {
-        return "ALTER TABLE " + childTable + " ADD CONSTRAINT " + childTable + "_" + parentTable + "_" + parentPrimaryKey + "_foreign FOREIGN KEY(" + childForeignKey + ") REFERENCES " + parentTable + "(" + parentPrimaryKey + ")"
+        return "ALTER TABLE " + childTable + " ADD CONSTRAINT " + childTable + "_" + parentTable + "_" + parentPrimaryKey + "_" + childForeignKey + "_foreign FOREIGN KEY(" + childForeignKey + ") REFERENCES " + parentTable + "(" + parentPrimaryKey + ")"
```

Comment:
A minor thing, but would it be easier to understand if we put the `childForeignKey` after `childTable`? Instead of having it ind the end. If you agree, remember to update `removeForeign(..)` as well.